### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by wwerner**

### CODE REVIEW

**Comment by gibbok** [comment]
@wwerner
what are we planning to have in this repository?
should we have this guide in our main sites?
thanks

**Comment by wwerner** [comment]
@gibbok, this is the implementation of the the external http connectivity package described in https://www.notion.so/actyx/API-Server-as-actyx-contrib-package-29d8eef5f0db4e508edb1f93c115da47

File: README.md [hunk 1]
```diff
@@ -2,21 +2,32 @@
...
+
+* ... add middlewares like body-parsers or authentication
+* ... add static file resources
+* ... provide additional routes or catch-all route handlers
 
+Additionally, you can query information from like your local source Id, the Pond state, swarm connectivity and whether the node running the HTTP connector is in sync with the swarm to be able to deal with error conditions better.
+
+## ⚖️ Trade-Offs
+
+Note, however, that web apps running on the HTTP connector do not provide the same level of resilience that Actyx Apps do. These applications are not distributed and run on a central server.
```
Comment (Alexander89) [inline README.md:28]: This is not always true. If the app is local it is still 100 % up

File: README.md [hunk 2]
```diff
@@ -27,30 +38,37 @@ npm install @actyx-contrib/actyx-http-connector
...
+
+In your `http-api` app's `index.ts`, import the HTTP connector, then use `httpConnector()` to create a server instance as shown below.
+For further details, please refer to the docs and the examples.
+
+```ts
+import { httpConnector, registryEntry } from '@actyx-contrib/actyx-http-connector'
 
-```typescript
-import { httpConnector, registryEntry } from '../../src'
-import { Pond } from '@actyx/pond'
```
Comment (Alexander89) [inline README.md:52]: Still required

**Pull request merged**